src/libLookup/Lookup.cpp

```diff
@@ -2093,46 +2084,60 @@ void Lookup::CommitTxBlocks(const vector<TxBlock>& txBlocks) {
 
   m_mediator.UpdateTxBlockRand();
 
-  if ((m_mediator.m_currentEpochNum % NUM_FINAL_BLOCK_PER_POW == 0) &&
-      (m_syncType != SyncType::NEW_LOOKUP_SYNC)) {
-    if (m_syncType == SyncType::RECOVERY_ALL_SYNC) {
-      LOG_EPOCH(
-          INFO, m_mediator.m_currentEpochNum,
-          "New node - At new DS epoch now, try getting state from lookup");
-      GetStateFromSeedNodes();
-    } else if (m_syncType == SyncType::LOOKUP_SYNC) {
-      LOG_EPOCH(INFO, m_mediator.m_currentEpochNum,
-                "Lookup node - Join back to network now.");
-
-      if (!m_currDSExpired) {
-        if (FinishRejoinAsLookup()) {
-          SetSyncType(SyncType::NO_SYNC);
-
-          if (m_lookupServer) {
-            if (m_lookupServer->StartListening()) {
-              LOG_GENERAL(INFO, "API Server started to listen again");
-            } else {
-              LOG_GENERAL(WARNING, "API Server couldn't start");
+  if (m_syncType == SyncType::NEW_SYNC || m_syncType == SyncType::NORMAL_SYNC) {
```

Comment:
The condition for `(m_syncType == SyncType::RECOVERY_ALL_SYNC` was removed. Just want to know what will happen in such case

Reply:
Good catch. However, the check condition is dormant. we should never come across `StartSynchronization` phase with m_syncType `RECOVERY_ALL_SYNC`. Also, `GetStateFromSeedNodes` should have been dormant long back. Lookup should not server complete state.